### PR TITLE
Remove `unplugin-icons`

### PR DESCRIPTION
**Pull request by qwerty287**
Instead, directly import icons.

### CODE REVIEW

File: web/src/components/user/UserGeneralTab.vue [hunk 1]
```diff
@@ -28,6 +28,8 @@ import Settings from '~/components/layout/Settings.vue';
 import { setI18nLanguage } from '~/compositions/useI18n';
 import { useTheme } from '~/compositions/useTheme';
 
+import InputField from '../form/InputField.vue';
```
Comment (anbraten) [inline web/src/components/user/UserGeneralTab.vue:31]:
```suggestion
import InputField from '~/components/form/InputField.vue';
```

**Review by anbraten — APPROVED** [review]

**Comment by anbraten** [comment]
From the docs its says you have to add this to the tsconfig:

```
// tsconfig.json
{
  "compilerOptions": {
    "types": [
      "unplugin-icons/types/vue",
    ]
  }
}
```

**Comment by anbraten** [comment]
Would actually try to stick to `iconify` as its pretty handy and flexibel. You can just change the import from `@iconify/json` to `@iconify-json/mdi`

**Comment by anbraten** [comment]
> I can revert it - but I don't really understand what you mean be "flexible". It's only flexible as long as you install all sets and that's explicitly what I not want to do

`unplugin-icons` has a quite unified way for importing different iconsets and seems to be well maintained in comparison to `@jamescoyle/vue-icon` (last update 3 years ago). I don't see a reason why we should replace it. 🤔 (Also really like the [index](https://icones.js.org/))

File: web/src/components/atomic/Button.vue [hunk 2]
```diff
@@ -28,7 +28,7 @@
           'bg-wp-control-error-200': color === 'red',
         }"
       >
-        <Icon name="loading" class="animate-spin" />
+        <Icon name="spinner" fill="#FFFFFF" />
```
Comment (6543) [inline web/src/components/atomic/Button.vue:31]: hardcoded collor?

**Review by lafriks — APPROVED** [review]

**Pull request merged**